src/vm/readytoruninfo.cpp

```diff
@@ -671,7 +671,32 @@ PCODE ReadyToRunInfo::GetEntryPoint(MethodDesc * pMD, BOOL fFixups /*=TRUE*/)
             m_entryPointToMethodDescMap.InsertValue(PCODEToPINSTR(pEntryPoint), pMD);
     }
```

Comment:
The call to JITCachedFunctionSearchStarted should be before the entrypoint gets inserted into the map - if we drop the code onto the floor, there is no point in inserting it into the map. I would actually put even before the FixupDelayList because of that's where it is done for NGen and because of there is no point in running the fixups if the code gets dropped either.